src/virt.rs

```diff
@@ -658,6 +658,7 @@ impl VirtContext {
         match cause {
             MCause::EcallFromSMode if policy.ecall_from_payload(mctx, self).overwrites() => {
                 // Nothing to do, the Policy module handles those ecalls
+                log::info!("Catching E-call from payload in the policy module");
```

Comment:
Let's keep it as a trace. You can also add some logging in your policy module and configure the logging to use trace for the policy module.